src/OSPSuite.Core/Domain/IProject.cs

```diff
@@ -7,6 +7,11 @@ namespace OSPSuite.Core.Domain
 {
    public interface IProject : IObjectBase, IWithChartTemplates
    {
+      /// <summary>
+      /// Defines the meta data associated with the project (when was it created, with which version etc...)
+      /// </summary>
+      CreationMetaData Creation { get; set; }
```

Comment:
Also in the project. That way we will know what was the version of the project when IT was created. Surprised we never thought about that before